### DISCUSSION TITLE
Higher z-index than 50 - Something like `z-full` or `z-max`

### DISCUSSION THREAD

**Discussion by jenstornell** [tailwindlabs/tailwindcss#2275] [Ideas]
Max `z-index` is `50`. (I know we can extend it ourselves)

When using external libraries, for example Leaflet (openstreetmap), they use a `z-index` of `800` for their zoom buttons.

Maximum `z-index` is [2147483647](https://stackoverflow.com/questions/491052/minimum-and-maximum-value-of-z-index) but it's not very common to go that high.

### Solution

Add something like `z-full` or `z-max` to make sure the element will be placed on top. That way you don't need to implement that many new numbers. It would still be enough to take care of these cases quite well.

**Comment by DCzajkowski** [comment]
The problem with `z-full` or `z-max` arises when you have several items with such class. In my experience using `z-index: 1` is enough in 99% of cases. It's a pity Leaflet uses high z-indices (I have experienced that myself with Leaflet, it sucks).

I think we should discourage a use of high index values, so 👎 from me. In those rare cases when you must use high indices, just add them to your config 🙂
